networkx/algorithms/centrality/group.py

```diff
@@ -2,8 +2,9 @@
 from copy import deepcopy
 
 import networkx as nx
+import pandas as pd
```

Comment:
Move the pandas and numpy imports inside the functions so that they don't get imported until the function is run instead of when it is defined.  The pypy test failed because that test does not have those packages available.

That means you'll also have to change the tests file.  For tests that call functions involving pandas or numpy, use the `pytest.importorskip` function to allow pytest to skip the test when the import doesn't work. For example

```
pytest.importorskip("pandas")
```
Or, if you need pandas in the test module:
```
pd = pytest.importorskip("pandas")
```